CADETProcess/transform.py

```diff
@@ -96,6 +96,14 @@ def __init__(
         self.allow_extended_input = allow_extended_input
         self.allow_extended_output = allow_extended_output
 
+
```

Comment:
Maybe it's sufficient to later check `if isinstance(transform, NormLinearTransform)` since it's probably gonna be the only linear transform.

Reply:
what about no transform? That's a linear transform mapping onto itself, I think.